service/lib/agama/software/manager.rb

```diff
@@ -70,6 +70,23 @@ class Manager # rubocop:disable Metrics/ClassLength
       GPG_KEYS_GLOB = "/usr/lib/rpm/gnupg/keys/gpg-*"
       private_constant :GPG_KEYS_GLOB
 
+      # location of the custom DUD package repository,
+      # see the /usr/lib/dracut/modules.d/99agama-dud/agama-dud-apply.sh script
+      DUD_REPOSITORY_DIR = "/var/lib/agama/dud/repo"
+      private_constant :DUD_REPOSITORY_DIR
+
+      # name for the custom DUD package repository, use some special name to
+      # minimize possible conflicts with user defined repositories, ugly name
+      # does not matter, it is deleted in the end anyway
```

Comment:
np: I think it is good enough. But we could even consider generating a random identifier (`agama-3a9d0ca` or whatever).

Reply:
Pkg-bindings can [already do that](https://github.com/yast/yast-pkg-bindings/blob/853496f527543e6d51730fd7e3126ad94b13c303/src/Source_Create.cc#L412-L419) if the alias is not specified or is empty.

But personally I'd rather avoid any randomness in the code, that can make things worse to reproduce. And it is easier to grep the logs if you know what you are looking for.

As we use only a single DUD repository even for multiple DUDs there is basically no risk for conflicts. (Unless some user really wants to shoot his leg and deliberately uses the same reserved name.)